scripts/core-triage/project.py

```diff
@@ -203,6 +203,16 @@ def main(
     # for each repo
     for repo in core_repos:
         print(f"Processing repository: {repo}...\n")
+        # private repos can't be connected.  Instead of bombing out, just skip them
+        try:
+            # TODO: add back label filtering
+            # get the list of PRs for the repo and label
+            prs = get_prs(repo, num_items)
+            # add PRs to the project
+            add_items_to_project(project_id, prs)
+        except:
```

Comment:
Should we catch a specific exception here? We went from bombing on all exceptions to continuing on all whereas the associated comment suggests we know why this might fail and how we want to handle that.

Also, I think we are only testing for the exception in the call to `get_prs`, not in the call to `add_items_to_project`. I know we would only want to add PRs to the project if there is no exception, but that feels like it should be in the `else` clause of the try block.

Reply:
This was a good point.  I realized we should just be always rejecting all private repos.  And then if it bombs out for some other reason we need to look into it - which not that a slack notification is sent we will know to do just that.